Resources/Prototypes/Reagents/toxins.yml

```diff
@@ -316,7 +316,15 @@
   plantMetabolism:
   - !type:PlantAdjustToxins
     amount: 10
-  # TODO hallucinations
```

Comment:
Keep the TODO in as these aren't actual hallucinations

Reply:
ok